queries.sql

```diff
@@ -1,17 +1,58 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
+BEGIN TRANSACTION;
 
-SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+  UPDATE animals SET species='unspecified';
 
-SELECT name FROM animals WHERE neutered=true AND escape_attempts<3;
+  SELECT * FROM animals ORDER BY id; 
 
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+ROLLBACK;
 
-SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
 
-SELECT * FROM animals WHERE neutered=true;
+BEGIN TRANSACTION;
 
-SELECT * FROM animals WHERE name!='Gabumon';
+  UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
 
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg<=17.3;
+  UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT TRANSACTION;
+
```

Comment:
- [ ] Just like I mentioned above, let try to verify that our transaction was really committed to the database.

```sql
SELECT species from animals;
```

Reply:
> * [x]  Just like I mentioned above, let try to verify that our transaction was really committed to the database.
> 
> ```sql
> SELECT species from animals;
> ```

- [ ] As the previous reviewer suggested, Please work on it. :+1: